grade_test.go

```diff
@@ -184,30 +187,31 @@ func TestCompile(t *testing.T) {
 
 func TestCompileWallSuccess(t *testing.T) {
 	cpp := `#include <iostream>
-	int main() { std::cout << "Hello world!" << std::endl; }`
+	int main() { std::cout << "Hello world!" << std::endl;
+	return 0; }`
```

Comment:
?

Reply:
That was straight up a ctrl+v, accidentally occurring twice. Going to fix on the next PR.